.gitignore

```diff
@@ -21,3 +21,6 @@
 /yarn-error.log
 
 .byebug_history
+
+#Ignore schemna file
+/db/schema.rb
```

Comment:
schema.rb is still there. Read about how to ignore a file which is already pushed to GIT